cmd/buddha.go

```diff
@@ -351,6 +351,9 @@ func executeHealthCheck(wg *sync.WaitGroup, cmd buddha.Command, check buddha.Che
 }
 
 func allFalse(arr []bool) bool {
+	if arr == nil {
```

Comment:
It's my understanding that the zero value of a slice (zero length) "acts" as `nil` (see: http://blog.golang.org/go-slices-usage-and-internals). I'm struggling to find precisely what this means in terms of whether it is internally the same thing, or whether they are equal in terms of equality. @jamescun?